README.md

```diff
@@ -431,6 +431,14 @@ Depends on configuration, but usually it is not needed to generate or use API ke
 ## 12. Registering new LLM provider
```

Comment:
Registering **a** new LLM provider (unrelated I know, but I saw it)

Reply:
fixed, thank you